src/OpenSSL/crypto.py

```diff
@@ -3010,6 +3010,14 @@ def load_pkcs7_data(type, buffer):
     return pypkcs7
 
 
+load_pkcs7_data = utils.deprecated(
+    load_pkcs7_data,
+    __name__,
+    "PKCS#7 support in pyOpenSSL is deprecated.",
```

Comment:
We should update this to say the same about cryptography's APIs once we merge support.

Reply:
Do you want to wait until PKCS7 is merged into pyca/cryptography then?

Reply:
Eeeh, I don't care too much. We'll get that merged pretty quickly so if you want to do it now that's fine.